apps/storybook/src/Selection.stories.tsx

```diff
@@ -36,38 +36,34 @@ const Template: Story<TemplateProps> = (args) => {
   const [selectedVectors, setSelectedVectors] = useState<[Vector2, Vector2]>();
 
   return (
-    <>
-      {selection && (
-        <p style={{ textAlign: 'center' }}>
-          {selectedVectors
-            ? `Selection from ${vectorToStr(
-                selectedVectors[0]
-              )} to ${vectorToStr(selectedVectors[1])}`
-            : 'No selection'}
-        </p>
```

Comment:
I noticed that the canvas would grow when opening the story, probably due to `display: grid` on the parent `FillHeight` decorator. So I now pass the current selection via the `title` prop of `VisCanvas`.